README.md

```diff
@@ -1 +1,65 @@
-# hello_microverse
\ No newline at end of file
+# hello_microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> One paragraph statement about the project.
+
```

Comment:
It will be great to add a small statement about this project. This will make your project even more professional. For example, _This is a project to learn how to professionally set up a project_.